client/types/account-overview.d.ts

```diff
@@ -10,7 +10,13 @@ export interface Account {
 	deposits_schedule: {
 		delay_days: number;
 		interval: string;
+		/**
```

Comment:
Not a concern for this PR, but I realise we should make this a union type to improve devex and help catch bugs.
```suggestion
		interval: 'manual' | 'daily' | 'weekly' | 'monthly';
```